lib/pymedphys/_dicom/ct/extend.py

```diff
@@ -122,35 +98,43 @@ def copy_slices_and_append(dicom_datasets, index_to_copy, number_of_slices):
         append(new_slice)
 
 
-def refresh_instance_numbers(dicom_datasets):
+def _refresh_instance_numbers(dicom_datasets):
     for i, dicom_dataset in enumerate(dicom_datasets):
         dicom_dataset.InstanceNumber = str(i)
 
 
-def generate_new_uids(dicom_datasets, uids=None):
+def _generate_new_uids(dicom_datasets, uids=None):
     if uids is None:
-        uids = generate_uids(len(dicom_datasets))
+        uids = _generate_uids(len(dicom_datasets))
 
     for dicom_dataset, uid in zip(dicom_datasets, uids):
         dicom_dataset.SOPInstanceUID = uid
```

Comment:
what about the Series Instance UID?
You've generated a new Series, you weren't the owner of the original Series Instance UID (probably) so you need to put these in a different Series just for that reason alone, but more practically, if you re-use the same series UID, a lot of systems may put the old ones and these new ones together and try to load them as a single volume.
In general a CT series is a single volume.  You can get in to "but I only changed the data by using a different reconstruction smoothing kernel, they are the same acquired sinogram data", but it doesn't help in terms of making the data more interoperable.

Reply:
Mis-named the commit as I was interrupted mid coding, nevertheless, here's the commit that fixes this one:

add6a8aa491c09fbb84d652f20c00804abf1bbae 

> You've generated a new Series, you weren't the owner of the original Series Instance UID (probably) so you need to put these in a different Series just for that reason alone

This was certainly an issue. Thank you for picking this one up.